embedded/store/ongoing_tx_options.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2022 Codenotary Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package store
+
+import (
+	"fmt"
+	"time"
+)
+
+type TxMode int
+
+const (
+	ReadOnlyTx TxMode = iota
+	WriteOnlyTx
+	ReadWriteTx
+)
+
+type TxOptions struct {
+	Mode TxMode
+	// SnapshotMustIncludeTxID is a function which receives the latest precommitted transaction ID as parameter.
+	// It gaves the possibility to require a snapshot which includes a percentage of the transactions already indexed.
```

Comment:
gives*

Just finding it a bit hard to understand this. I understood from our conversation on what this does, but would be simpler if you can simplify this sentence  

```
t gaves the possibility to require a snapshot which includes a percentage of the transactions already indexed.
```